library/ssl_tls13_server.c

```diff
@@ -2640,6 +2641,10 @@ static int ssl_tls13_prepare_new_session_ticket( mbedtls_ssl_context *ssl,
 
     MBEDTLS_SSL_DEBUG_MSG( 2, ( "=> prepare NewSessionTicket msg" ) );
 
+    if( ssl->handshake->resume == 1 )
```

Comment:
Why only one ticket in case of a session resumption ?

Reply:
I am try to resolve `Client Tracking Prevention`. For now, I agree with you, I think we should not think about the issue in this PR.

```
C.4.  Client Tracking Prevention

   Clients SHOULD NOT reuse a ticket for multiple connections.  Reuse of
   a ticket allows passive observers to correlate different connections.
   Servers that issue tickets SHOULD offer at least as many tickets as
   the number of connections that a client might use; for example, a web
   browser using HTTP/1.1 [[RFC7230](https://datatracker.ietf.org/doc/html/rfc7230)] might open six connections to a
   server.  Servers SHOULD issue new tickets with every connection.
   This ensures that clients are always able to use a new ticket when
   creating a new connection.
```